src/main/java/pl/jablonskanycz/bakery/database/services/AddressService.java

```diff
@@ -36,8 +36,7 @@ private AddressEntity returnAddressIfExists(Long addressToUpdateId) {
         return addressRepository.findById(addressToUpdateId).orElseThrow(() -> new RuntimeException("Address not found"));
     }
 
-//    TODO
-//    public void deleteAddress(Long addressToDeleteId){
-//        addressRepository.delete(returnAddressIfExists(addressToDeleteId));
-//    }
+    public void deleteAddress(Long addressToDeleteId){
+        addressRepository.delete(returnAddressIfExists(addressToDeleteId));
```

Comment:
When we are deleting objects there exist mainly 2 scenarios that can happen:
1. The object to delete exists in the database and therefore the deletion should work correctly
2. Or the object doesn't exist in the database in the first place so there is nothing to delete.

From there, there are 2 questions to answer:
How would you like the 2nd scenario to be handled?
And then: how is it being handled currently :) ?

_In my opinion, before attempting deletion we should check if the object existed before and only if it then proceed to deletion. If it didn't, then do not do anything. What do you think?_

Let's adjust the code accordingly to your answers to the above questions.

Reply:
I see you updated the code but I see the outcome is still the same, actually. That is, it still throws an exception when provided an address that doesn't exist. 
<img width="980" alt="image" src="https://github.com/user-attachments/assets/971ed808-62ae-44b0-b191-1dadd03a3f9b">
I am wondering, is it going according to how you wanted it to behave :)?

Reply:
OK. Got it. 
Yes, in the beginning that was my intention to throw an exception. But now I've changed it. I'm not sure if I'm operating correctly on Optionals but it works :)

Reply:
you are :) good!